catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -175,18 +204,19 @@ private MetacardImpl initializeMetacard(String id, String contentInput) {
                     .flatMap(Collection::stream)
                     .collect(Collectors.toSet());
 
-            metacard = new MetacardImpl(new MetacardTypeImpl(BasicTypes.BASIC_METACARD.getName(),
-                    BasicTypes.BASIC_METACARD,
+            metacard = new MetacardImpl(new MetacardTypeImpl(metacardType.getName(),
```

Comment:
In general we should drop attributes that don't map into the taxonomy.  We should _try_ to map fields if possible though.  If we think there is important information that can't be mapped, we should discuss this and potentially add them either as ext attributes or as new additions to the taxonomy itself.